tests/core/pyspec/eth2spec/utils/ssz/ssz_impl.py

```diff
@@ -1,157 +1,10 @@
-from ..merkle_minimal import merkleize_chunks
-from ..hash_function import hash
-from .ssz_typing import (
-    SSZValue, SSZType, BasicValue, BasicType, Series, Elements, Bits, boolean, Container, List, ByteList,
-    Bitlist, Bitvector, uint, Bytes32
-)
+from remerkleable.core import View
+from remerkleable.byte_vector import Bytes32
 
-# SSZ Serialization
-# -----------------------------
 
-BYTES_PER_LENGTH_OFFSET = 4
+def serialize(obj: View) -> bytes:
```

Comment:
I'm hesitant to fully delete the existing minimal impl, especially because it is (likely) much easier to build and test for correctness than remerkable. 

When SSZ in own repo, do you see value in maintaining this minimal impl there?

Reply:
Legacy code has its own costs. Also see https://github.com/protolambda/remerkleable/blob/master/remerkleable/test_impl.py for hand-crafted tests. Can't be more correct than that. 

Only value would be in comparing the two implementations, but I'd rather compare against the old test vectors, and move on to new testing (more handcrafted edge cases, and compare against clients, fuzzing, etc.) instead of maintaining legacy code.